chroma-manager/chroma_core/lib/service_config.py

```diff
@@ -318,17 +299,28 @@ def _start_services(self):
                 log.error(error)
                 raise RuntimeError(error)
 
-        SUPERVISOR_START_TIMEOUT = 10
+        xs = self.CONTROLLED_SERVICES + self.MANAGER_SERVICES
+
+        SYSTEMD_START_TIMEOUT = 10
         t = 0
         while True:
-            if not SupervisorStatus().get_non_running_services():
+            service_config = self._service_config(xs)
+
+            not_running = []
+
+            for x in xs:
+                service_status = service_config[x]
+                if not service_status['running']:
+                    not_running.append(x)
+
+            if len(not_running) == 0:
                 break
             else:
                 time.sleep(1)
                 t += 1
-                if t > SUPERVISOR_START_TIMEOUT:
+                if t > SYSTEMD_START_TIMEOUT:
```

Comment:
Will it succeed after > 1 tries if it fails the first time?

If so, that seems like something that needs to be fixed.  Don't want to mask issues here and now that we are putting all of this under the control of `systemd` it seems like a fine time to discover if there are issues which running > 1 times works around.

Reply:
Will fix.